src/api/helpers/formatEntityData.js

```diff
@@ -130,10 +130,30 @@ export function getEntityIdentifiers(entity: object) {
 		};
 }
 
+
 export function getEntityRelationships (entity: object) {
 	return _.isNil(entity) ? null :
 		{
 			bbid: _.get(entity, 'bbid', null),
-			relationships: _.get(entity, 'relationshipSet.relationships', null)
+			relationships: _.get(entity, 'relationshipSet.relationships', []).map((relationship) => {
+				return {
+					id: relationship.id,
+					targetBbid: entity.bbid === relationship.sourceBbid ?
```

Comment:
`entity.bbid === relationship.sourceBbid`
Same code repeated four times? Time to refactor :)